src/AccessControl/tests/testZopeGuards.py

```diff
@@ -144,6 +144,23 @@ def test_attr_handler_table(self):
         finally:
             ContainerAssertions[_dict] = old
 
+    def test_simple_object_policies(self):
+        '''
+        Test that we are able to access attributes of simple types (here:
+        unicode and bytes)
+        '''
+        from AccessControl.ZopeGuards import guarded_getattr
+        orig_value = self.__sm.reject
+        self.__sm.reject = True
```

Comment:
This should be done in a `try-finally` block to other tests running after this one do not break if this test fails.